bloom-housing/bloom
# feat: add Programs section to listings management

# Pull Request Template

## Issue Overview

This PR addresses #1868

## Description

Adds the Programs section to listings management. It's identical to the Preferences section. Also fixes an existing bug with Preferences where if you opened the drawer, deleted an already select preference from the draggable list, then opened the checkboxes, the deleted checkbox would still be checked.

Still waiting to hear from Kathy what the titles of the Programs should be in LM since we are currently showing just the questions, and have no titles. Will update seeds when I get those.

## How Can This Be Tested/Reviewed?

Add, remove, and reorder both preferences and programs. Ensure they appear on the detail page as well.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
